subsys/nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -1,353 +1,1120 @@
+config PSA_WANT_ECC_SECT_K1_233
 	bool
-	prompt "PSA stream cipher support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_STREAM_CIPHER && \
-		     !PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_CC3XX
+	depends on PSA_ACCEL_KEY_PAIR_K233
 
-config PSA_CRYPTO_DRIVER_ECC_MONTGOMERY_255_OBERON
+config PSA_WANT_ECC_SECT_K1_239
 	bool
-	prompt "PSA ECC Curve25519 support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_MONTGOMERY_255 && \
-		     !PSA_CRYPTO_DRIVER_ECC_MONTGOMERY_255_CC3XX
+	depends on PSA_ACCEL_KEY_PAIR_K239
 
-config PSA_CRYPTO_DRIVER_ECC_SECP_R1_224_OBERON
+config PSA_WANT_ECC_SECT_K1_283
 	bool
-	prompt "PSA ECC secp224r1 - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_SECP_R1_224 && \
-		     !PSA_CRYPTO_DRIVER_ECC_SECP_R1_224_CC3XX
+	depends on PSA_ACCEL_KEY_PAIR_K283
 
-config PSA_CRYPTO_DRIVER_ECC_SECP_R1_256_OBERON
+config PSA_WANT_ECC_SECT_K1_409
 	bool
-	prompt "PSA ECC secp256r1 - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_SECP_R1_256 && \
-		     !PSA_CRYPTO_DRIVER_ECC_SECP_R1_256_CC3XX
+	depends on PSA_ACCEL_KEY_PAIR_K409
 
-config PSA_CRYPTO_DRIVER_ECC_SECP_R1_384_OBERON
+config PSA_WANT_ECC_SECT_K1_571
 	bool
-	prompt "PSA ECC secp384r1 - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_SECP_R1_384 && \
-		     !PSA_CRYPTO_DRIVER_ECC_SECP_R1_384_CC3XX
+	depends on PSA_ACCEL_KEY_PAIR_K571
 
-config PSA_CRYPTO_DRIVER_ECC_TWISTED_EDWARDS_255_OBERON
+config PSA_WANT_ECC_SECT_R1_163
 	bool
-	prompt "PSA ECC Curve Ed25519 support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_TWISTED_EDWARDS_255
+	depends on PSA_ACCEL_KEY_PAIR_R163
 
-config PSA_CRYPTO_DRIVER_RSA_KEY_SIZE_1024_OBERON
+config PSA_WANT_ECC_SECT_R1_233
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_R233
+
+config PSA_WANT_ECC_SECT_R1_283
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_R283
+
+config PSA_WANT_ECC_SECT_R1_409
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_R409
+
+config PSA_WANT_ECC_SECT_R1_571
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_R571
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_160
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB160
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_192
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB192
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_224
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB224
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_320
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB320
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_384
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB384
+
+config PSA_WANT_ECC_BRAINPOOL_P_R1_512
+	bool
+	depends on PSA_ACCEL_KEY_PAIR_PB512
+
+config PSA_WANT_KEY_TYPE_ARIA
+	bool
+	depends on PSA_ACCEL_ARIA
+
+config PSA_WANT_KEY_TYPE_CAMELLIA
+	bool
+	depends on PSA_ACCEL_CAMELLIA
+
+config PSA_WANT_KEY_TYPE_DES
+	bool
+	depends on PSA_ACCEL_DES
+
+
+# Set all mbedTLS algorithms to accelerated to disable all mbedTLS fallback functions
+
+config MBEDTLS_PSA_ACCEL_ALG_CBC_NO_PADDING
```

Comment:
You are right,  checking it more thoroughly most of the symbols are no even converted to header configs. So even if config_psa.h was included this will not do much.
Anyway, I will remove them.